package-lock.json

```diff
@@ -10407,6 +10407,198 @@
 				"postcss-syntax": ">=0.36.2"
 			}
 		},
+		"node_modules/@swc/core": {
+			"version": "1.4.8",
+			"resolved": "https://registry.npmjs.org/@swc/core/-/core-1.4.8.tgz",
+			"integrity": "sha512-uY2RSJcFPgNOEg12RQZL197LZX+MunGiKxsbxmh22VfVxrOYGRvh4mPANFlrD1yb38CgmW1wI6YgIi8LkIwmWg==",
+			"hasInstallScript": true,
+			"dependencies": {
+				"@swc/counter": "^0.1.2",
+				"@swc/types": "^0.1.5"
+			},
+			"engines": {
+				"node": ">=10"
+			},
```

Comment:
I installed this locally on your branch and didn't have any issues, but I'm curious if the incongruity here will cause an issue with the app using `"npm": ">=8.0.0"`.

Reply:
Great question! The `engines` defined in our `package.json` file is a subset of this `engines`, so it won't cause any issues. You can think of this one as being equivalent to:

```
"engines": {
  "node": ">=10",
  "npm": "*"
}
```

Meaning that _any_ version of npm is acceptable.

Reply:
Thanks for explaining! Much appreciated